### PR TITLE
INTMDB-212: Deprecation of Project IP Whitelist

### PR DESCRIPTION
**Pull request by coderGo93**
## Description

- Deleted resources, datasources and its docs for Project Whitelist
- Added guide doc for deprecation project whitelist and import project access list
- Added one terratest to show that it works by creating one project whitelist in v0.9.1 and then import with project access list in another folder and the re plan show not changes

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: website/docs/guides/1.0.0-upgrade-guide.html.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.0.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-100-upgrade-guide"
+description: |-
+    MongoDB Atlas Provider 0.1.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.0.0: Upgrade and Information Guide
+
+## Deprecation of Project IP Whitelist and import with Project IP Access List
```
Comment (themantissa) [inline website/docs/guides/1.0.0-upgrade-guide.html.markdown]: We already deprecated it so this is the EoL, perhaps better said as:
 ## End of Life and removal of Project IP Whitelist and import with Project IP Access List

**Review by themantissa — CHANGES_REQUESTED** [review]
One small change in terms but otherwise lgtm!

**Review by themantissa — APPROVED** [review]
LGTM!

**Review by leofigy — APPROVED** [review]
LGTM :)

**Pull request merged**